site/styles/_shopping.scss

```diff
@@ -13,11 +13,9 @@
 
     .s72-btn-close {
       background: transparent;
-      color: var(--body-color);
```

Comment:
Removing this means the other modals have a black X on a black background (just tested it).  I don't think you should remove the colour and opacity lines from this file.

Also, is there a reason why the share modal is light theme when the rest of the modals on the site are dark theme?

Reply:
Good find @PeculiarGoat. I've re-added that line and the x has reappeared.

I had pushback from design on this, so had to revert it.

I really preferred dark mode but picking battles.